src/System.Globalization/tests/CultureInfo/CultureInfoTests.netstandard1.7.cs

```diff
@@ -0,0 +1,122 @@
+            Assert.Equal(ci.LCID, lcid);
+            Assert.Equal(ci.UseUserOverride, false);
+            Assert.Equal(ci.IsReadOnly, true);
+
+            ci = CultureInfo.GetCultureInfo(cultureName, "");
+            Assert.Equal(ci.Name, cultureName);
+            Assert.Equal(ci.LCID, lcid);
+            Assert.Equal(ci.UseUserOverride, false);
+            Assert.Equal(ci.IsReadOnly, true);
+            Assert.Equal(ci.TextInfo, CultureInfo.InvariantCulture.TextInfo);
+            Assert.Equal(ci.CompareInfo, CultureInfo.InvariantCulture.CompareInfo);
+
+            ci = CultureInfo.CreateSpecificCulture(cultureName);
+            Assert.Equal(ci.Name, specificCultureName);
+
+            ci = CultureInfo.GetCultureInfoByIetfLanguageTag(cultureName);            
+            Assert.Equal(ci.Name, cultureName);
+        }
+
+        [Fact]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        public void InstalledUICultureTest()
+        {
+            // as we didn't change current UI culture, Installed UI culture should match current UI culture
+            Assert.Equal(CultureInfo.InstalledUICulture, CultureInfo.CurrentUICulture);
+        }
+
+        [Fact]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        public void GetCulturesTest()
+        {
+            foreach (CultureInfo ci in CultureInfo.GetCultures(CultureTypes.NeutralCultures))
+                Assert.True(ci.IsNeutralCulture || ci.Equals(CultureInfo.InvariantCulture));
+
+            foreach (CultureInfo ci in CultureInfo.GetCultures(CultureTypes.SpecificCultures))
+                Assert.False(ci.IsNeutralCulture);
+        }
+
+        [Fact]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        public void ClearCachedDataTest()
+        {
+            CultureInfo ci = CultureInfo.GetCultureInfo("ja-JP");
+            Assert.True((object) ci == (object) CultureInfo.GetCultureInfo("ja-JP"));
+            ci.ClearCachedData();
+            Assert.False((object) ci == (object) CultureInfo.GetCultureInfo("ja-JP"));
+        }
+
+        [Fact]
+        [PlatformSpecific(Xunit.PlatformID.Windows)] // Non Windows is not supported yet
+        public void CultureNotFoundExceptionTest()
+        {
+            Assert.Throws<CultureNotFoundException>("name", () => new CultureInfo("!@#$%^&*()"));
+            Assert.Throws<CultureNotFoundException>("name", () => new CultureInfo("This is invalid culture"));
+            Assert.Throws<CultureNotFoundException>("name", () => new CultureInfo("longCulture" + new string('a', 100)));
+            Assert.Throws<CultureNotFoundException>("culture", () => new CultureInfo(0x1000));
+
+            try { new CultureInfo("This is invalid culture"); }
+            catch (CultureNotFoundException e) { Assert.Equal("This is invalid culture", e.InvalidCultureName); }
```

Comment:
Assert.Throws returned thrown exception, so instead of doing this try/catch, you can just augment the case of this above:
```C#
CultureNotFoundException e = Assert.Throws<CultureNotFoundException>("name", () => new CultureInfo("This is invalid culture"));
Assert.Equal("This is invalid culture", e.InvalidCultureName);
```